samples/IISSample/project.json

```diff
@@ -10,14 +10,23 @@
   },
   "frameworks": {
     "net451": { },
-    "netstandardapp1.5": {
+    "netcoreapp1.0": {
       "dependencies": {
-          "NETStandard.Library": "1.5.0-*"
+        "Microsoft.NETCore.App": {
+          "version": "1.0.0-*",
+          "type": "platform"
         },
+      },
       "imports": "dnxcore50"
     }
   },
-  "content": [
-    "web.config"
-  ]
+  "tools": {
+    "Microsoft.AspNetCore.Server.IISIntegration.Tools": {
```

Comment:
We can't add the tool to samples or tests in this repo, you can't use tools when you have a source reference to it.